src/Twig/PhoneFormatExtension.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace App\Twig;
+
+use Twig\Extension\AbstractExtension;
+use Twig\TwigFilter;
+use Twig\TwigFunction;
+
+class PhoneFormatExtension extends AbstractExtension
+{
+    public function getFilters(): array
+    {
+        return [
+            // If your filter generates SAFE HTML, you should add a third
+            // parameter: ['is_safe' => ['html']]
+            // Reference: https://twig.symfony.com/doc/2.x/advanced.html#automatic-escaping
+            new TwigFilter('formatPhoneFr', [$this, 'formatPhoneFr']),
+        ];
+    }
+
+    public function formatPhoneFr($value)
```

Comment:
ok mais 
1. si tu veux imposer ce format de phone number, il faut etre sur et certain du format en entrée (là tu n'as que celui de Peter donc éventuellement tu peux imposer un format en entrée... pour des utilisateurs ça serait plus compliqué.
2. ton code est améliorable. 
Une fois que tu as fait ton str_split tous les 2 chiffres, utilise simplement implode plutot que de refaire une boucle for

Reply:
fait